apps/app/components/core/modals/image-upload-modal.tsx

```diff
@@ -44,6 +44,9 @@ export const ImageUploadModal: React.FC<Props> = ({
   }, []);
 
   const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept: {
+      "image/*": [".jpeg", ".png", ".jpg", ".webp"],
+    },
     onDrop,
```

Comment:
Can you also add a small message informing the user about the type of files accepted?

Reply:
Sure